upup/pkg/fi/cloudup/awstasks/launchtemplate_target_terraform.go

```diff
@@ -187,6 +189,8 @@ func (t *LaunchTemplate) RenderTerraform(target *terraform.TerraformTarget, a, e
 		MetadataOptions: &terraformLaunchTemplateInstanceMetadata{
 			HTTPTokens:              e.HTTPTokens,
 			HTTPPutResponseHopLimit: e.HTTPPutResponseHopLimit,
+			// see issue https://github.com/hashicorp/terraform-provider-aws/issues/12564.
```

Comment:
```suggestion
			// See issue https://github.com/hashicorp/terraform-provider-aws/issues/12564.
```

Reply:
Consider moving this up to the top of the struct initializer, reordering these three fields so that they match the struct definition at line 113.

Reply:
yup will do.